examples/README.md

```diff
@@ -177,7 +177,7 @@ You can configure Sparkling Water using the following variables:
 13. Generate an R-code producing residual plot:
   ```scala
   import org.apache.spark.examples.h2o.AirlinesWithWeatherDemo2.residualPlotRCode
-  residualPlotRCode(predictionH2OFrame, 'predict, bigTable, 'ArrDelay)  
+  residualPlotRCode(predictionH2OFrame, 'predict, bigTable, 'ArrDelay, h2oContext)  
```

Comment:
Because if someone is running regular h2o at the same time they are trying this sparkling water example, then the residual plot r code won't execute.